internal/app/tfsec/rules/azure/securitycenter/set_required_contact_details_email_rule.go

```diff
@@ -0,0 +1,71 @@
+package securitycenter
+
+// generator-locked
+import (
+	"github.com/aquasecurity/tfsec/pkg/result"
+	"github.com/aquasecurity/tfsec/pkg/severity"
+
+	"github.com/aquasecurity/tfsec/pkg/provider"
+
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/block"
+
+	"github.com/aquasecurity/tfsec/pkg/rule"
+
+	"github.com/aquasecurity/tfsec/internal/app/tfsec/scanner"
+)
+
+func init() {
+	scanner.RegisterCheckRule(rule.Rule{
```

Comment:
Hey @brandon-bailey  - we should just enrich https://github.com/aquasecurity/tfsec/blob/master/internal/app/tfsec/rules/azure/securitycenter/set_required_contact_details_rule.go with email - its essentially the same check

Reply:
That makes sense, I was just hesitant to modify existing rules. I have removed this check in this PR and will brainstorm the best way to include it in the existing `set_required_contact_details_rule`

Reply:
I think just chuck it in below the phone one and use the tests you'd already created will be enough.

so result for no phone, result for no email or both results. that should suffice